tests/foreman/ui/test_dashboard.py

```diff
@@ -0,0 +1,197 @@
+"""Test module for Dashboard UI"""
+from robottelo.decorators import stubbed, tier3
+from robottelo.test import UITestCase
+
+
+class DashboardTestCase(UITestCase):
+    """Tests for Dashboard UI"""
+
+    @stubbed()
+    @tier3
+    def test_positive_save(self):
+        """@Test: Save the Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Content Dashboard
+        2.Try to remove some widgets
+        3.Select the Manage Dropdown box
+        4.Save the Dashboard
+
+        @Assert: Dashboard Widgets are saved successfully.
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_reset(self):
+        """@Test: Reset the Dashboard to default UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Content Dashboard
+        2.Try to remove some widgets
+        3.Select the Manage Dropdown box
+        4.Save the Dashboard
+        5.Dashboard Widgets are saved successfully
```

Comment:
@sthirugn the notification message reads: "Widget positions successfully saved." so perhaps that could be an appropriate description too.